infra/notifications/src/app.js

```diff
@@ -62,18 +62,18 @@ const config = {
   verbose: args['--verbose'] || false
 }
 
+logger.log('Configuration:')
 logger.log(config)
 
 // ------------------------------------------------------------------
 
-// should be tightened up for security
+// TODO: Should be tightened up for security
```

Comment:
When deployed this is all handled by nginx, so this should only be applied in development, i.e. `NODE_ENV === 'development'`